backend/src/Query.js

```diff
@@ -0,0 +1,12 @@
+const Query = {
+  user(context, info) {
+    if (!context.request.userId) {
```

Comment:
I didn't find property `request` in our context. Would suggest to go with something like `if (!args.id)`